package.json

```diff
@@ -36,33 +39,41 @@
     "eth-rpc-errors": "^4.0.2"
   },
   "devDependencies": {
-    "@lavamoat/allow-scripts": "^1.0.5",
-    "@metamask/auto-changelog": "^3.0.0",
-    "@metamask/eslint-config": "^9.0.0",
-    "@metamask/eslint-config-jest": "^9.0.0",
-    "@metamask/eslint-config-nodejs": "^9.0.0",
-    "@metamask/eslint-config-typescript": "^9.0.1",
-    "@types/jest": "^26.0.13",
-    "@types/node": "^17.0.23",
-    "@typescript-eslint/eslint-plugin": "^4.21.0",
-    "@typescript-eslint/parser": "^4.21.0",
-    "eslint": "^7.23.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.3.4",
-    "eslint-plugin-jsdoc": "^36.1.0",
+    "@lavamoat/allow-scripts": "^2.3.0",
+    "@metamask/auto-changelog": "^3.1.0",
+    "@metamask/eslint-config": "^11.0.1",
+    "@metamask/eslint-config-jest": "^11.0.0",
+    "@metamask/eslint-config-nodejs": "^11.0.1",
+    "@metamask/eslint-config-typescript": "^11.0.0",
+    "@types/jest": "^29.5.0",
+    "@types/node": "^18.15.11",
+    "@typescript-eslint/eslint-plugin": "^5.43.0",
+    "@typescript-eslint/parser": "^5.43.0",
+    "depcheck": "^1.4.3",
+    "eslint": "^8.27.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.1.5",
+    "eslint-plugin-jsdoc": "^39.6.2",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "jest": "^27.5.1",
-    "prettier": "^2.2.1",
-    "prettier-plugin-packagejson": "^2.2.11",
-    "rimraf": "^3.0.2",
-    "ts-jest": "^27.1.4",
-    "ts-node": "^10.7.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "jest": "^29.5.0",
+    "jest-it-up": "^2.2.0",
+    "prettier": "^2.8.7",
+    "prettier-plugin-packagejson": "^2.4.3",
+    "rimraf": "^5.0.0",
+    "ts-jest": "^29.1.0",
+    "ts-node": "^10.9.1",
+    "typedoc": "^0.24.1",
     "typescript": "~4.8.4"
   },
+  "packageManager": "yarn@3.5.0",
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=16.0.0"
```

Comment:
Should we hold off on bumping this? Mobile still uses Node 14, and we haven't bumped the Node version across libraries yet (I'm guessing for that reason). Although it sounds like based on a thread in #metamask-dev that mobile is working on this. @Gudahtt do you know anything about this? Also forgive me if there's context I'm missing for this PR.

Reply:
We want to start bumping all packages to Node 16 since 14 is EOL at the end of April. 

Since this package is rarely updated it seems fine to me to update it even if mobile can't use the changes instantly (bumping `core` for example would be a different story). Regardless though, from what Mark told me, mobile is a few weeks away from having Node 16.

Reply:
Ah okay. Gotcha. That makes sense then. Just wanted to make sure no one was surprised by this. I agree that this package isn't updated that often. We are about to replace `web3-provider-engine` with `json-rpc-engine` and friends in `core`, but we can certainly use the existing version for the time being as it does serve our purpose fine and then upgrade later when we bump all the `core` packages to Node 16.